### PR TITLE
Add petreeftime to codeowners

### PR DESCRIPTION
**Pull request by alindima**
Signed-off-by: alindima <alindima@amazon.com>

### CODE REVIEW

File: CODEOWNERS
Comment (lauralt) [inline CODEOWNERS:3]: Can we add Petre on the same line with you (at least for consistency with the other repos)? The github documentation seems to recommend the same thing: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners.

Comment (alindima) [reply CODEOWNERS]: sure, done

**Review by lauralt — APPROVED** [review]

**Review by petreeftime — APPROVED** [review]

**Review by andreeaflorescu — APPROVED** [review]

**Comment by lauralt** [comment]
Hmm, looks like we can merge PRs in this repository without waiting for the CI to pass.

**Comment by lauralt** [comment]
Updated the status checks with buildkite/seccompiler-ci.

**Pull request merged**
